doc/extdev/builderapi.rst

```diff
@@ -13,6 +13,7 @@ Builder API
 
    .. graphviz:: /_static/diagrams/sphinx_build_flow.dot
       :caption: UML for the standard Sphinx build workflow
```

Comment:
```suggestion
      :caption: Call graph for the standard Sphinx build workflow
```

Nitpick found while reading: I believe this is not a [formal UML diagram](https://en.wikipedia.org/wiki/Unified_Modeling_Language#Diagrams), but a [call graph](https://en.wikipedia.org/wiki/Call_graph).

Reply:
done 👍